components/02-elements/minicart/minicart.hbs

```diff
@@ -0,0 +1,77 @@
+<a class="action showcart minicart-link"
+   href="#"
```

Comment:
And it should be `button`, since something with `href="#"` isn't a link anymore.